.github/workflows/ci.yml

```diff
@@ -8,11 +8,12 @@ jobs:
       matrix:
         ruby-version: [3.2, 3.1, '3.0', 2.7]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up Ruby ${{ matrix.ruby-version }}
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
```

Comment:
I don't feel strongly about this existing. There are only 25 specs and four gem dependencies